### PR TITLE
Player reload fix

### PR DESCRIPTION
**Pull request by jatinkatyal13**

### CODE REVIEW

**Comment by abhishek97** [comment]
This pull request **introduces 1 alert** when merging b53b64c3204d5abb367dcd32d3d665cc0b808711 into 52a4ce10bbc175f4dea15ca04f80181e33d950ad - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/codingblocks.online.projectx/rev/pr-11eb6ecf7b406c9f1b38621eb08fcf6ed406a238)

**new alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

File: app/pods/attempt/old-content/route.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+import Route from '@ember/routing/route';
+
+export default class AttemptOldContentRoute extends Route {
+  beforeModel(transition) {
+    debugger
```
Comment (abhishek97) [inline app/pods/attempt/old-content/route.js]: isko hata lo

Comment (jatinkatyal13) [reply app/pods/attempt/old-content/route.js]: Done ! 😄

**Comment by abhishek97** [comment]
This pull request **introduces 1 alert** when merging ffb8a3e41d1521df93555b88d484c4f2bd2e2e77 into 24bf71cf5658988794ea134d3353c4547fd94bff - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/codingblocks.online.projectx/rev/pr-acbf436c6416d23cf655392673f1059107188f88)

**new alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by abhishek97** [comment]
This pull request **introduces 1 alert** when merging aa501eb655769496c7742bbd53da90004b8acbe1 into 24bf71cf5658988794ea134d3353c4547fd94bff - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/codingblocks.online.projectx/rev/pr-6ce03b69e6b59cefda005df5f492818e963d78ce)

**new alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Comment by abhishek97** [comment]
This pull request **introduces 1 alert** when merging aa501eb655769496c7742bbd53da90004b8acbe1 into b532f9975a4e1fedb943ce061061e41233879a2d - [view on LGTM.com](https://lgtm.com/projects/g/coding-blocks/codingblocks.online.projectx/rev/pr-7c3f671a658011a65757af20e3042fced53d2387)

**new alerts:**

* 1 for Unused variable, import, function or class

---

*Comment posted by [LGTM.com](https://lgtm.com)*

**Pull request merged**